### ISSUE TITLE
LMS text readability issues

### ISSUE THREAD

**Issue by sandroscosta** [fccn/openedx-platform#16]
AMAs accessibility team sent us a report, after reviewing the new course run content.
They have suggested some improvements:

* Links in the middle of the text are only supported in color. Color-blind people won't be able to see where the link is. People with low vision will find it very difficult. Links should also be distinguished by their form (the most common form is underlined). This change should be made to the NAU CSS so that it affects all the links in the body of the text. 

![Image](https://github.com/fccn/edx-platform/assets/34194007/e4e703c1-f84e-4165-8f54-6a1a51cd44b3)

* The font size for body text is 13px. The Usability and Accessibility Seal requirements recommend a minimum of 12pt (i.e. 16px). Is it possible to change this in CSS?

**Comment by sandroscosta** [comment]
I did some checking and the default site font size is 16px, so this argument is not valid.

![Image](https://github.com/fccn/edx-platform/assets/34194007/f9355c1f-0de8-423b-8947-3a3e12ddaf4a)

Also gathered that the team at AMA was redefining the html head and styles, subverting the way content is displayed. I've alerted the team about this.

![Image](https://github.com/fccn/edx-platform/assets/34194007/2a2bee52-407f-4c0e-aee6-13692236c303)

**Comment by igobranco** [comment]
@sandroscosta  so this is a content problem of that course, and not a general problem of the platform, right?
If, yes, please document it on Course team wiki, on things that they shouldn't be doing, I would suggest on the accessibility course content topic.

**Comment by sandroscosta** [comment]
Documented https://plataforma-nau.atlassian.net/wiki/spaces/NN/pages/2853666823/Conte+dos+HTML

**Comment by igobranco** [comment]
@sandroscosta this issue wasn't closed! Because it wasn't installed on production!

**Comment by igobranco** [comment]
Deployed in PROD! Working as expected

**Comment by igobranco** [comment]
Course link in PROD: https://lms.nau.edu.pt/courses/course-v1:AMA+AFAW_BP+2024_T2/courseware/94136c76e9c141d684d389994fb9e4c8/07111a315d3048b48dc12d9dde7c2dac/?activate_block_id=block-v1%3AAMA%2BAFAW_BP%2B2024_T2%2Btype%40sequential%2Bblock%4007111a315d3048b48dc12d9dde7c2dac
